lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -32,6 +33,12 @@ def is_enabled?(params = {}, context = nil)
 
       private
 
+      def context_invalid?(stickiness, context)
+        return false if ['random', 'default'].include?(stickiness)
```

Comment:
```suggestion
        # Context is always valid (and ignored) when stickiness is default or random, as context is not really relevant then.
        return false if ['random', 'default'].include?(stickiness)
```

If you don't want to add the comment here, feel free to add the equivalent in the squashed commit message.

Reply:
ill add it to the commit message, the tests shows this as well